client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -280,61 +280,29 @@ export class DisplayTable extends React.Component {
       .reverse()
       .value();
 
-    const {
-      paginate_by,
-      paginated_data,
-      change_paginate_by,
-      show_pagination_controls,
-      page_selector,
-    } = (() => {
-      const paginate_by =
-        size_to_paginate && _.size(sorted_filtered_data) > size_to_paginate
-          ? state_paginate_by
-          : 0;
-
-      const num_pages_exact = paginate_by
-        ? _.size(sorted_filtered_data) / paginate_by
-        : -1;
-
-      const num_pages =
-        num_pages_exact >= 0 && _.isInteger(num_pages_exact) //if we have the exact amount of items to fill an entire page, it will start a new empty page
-          ? num_pages_exact - 1 //thus we want to get rid of the extra page (as it has no content)
-          : _.toInteger(num_pages_exact); //start counting from 0 (-1 from the literal amount of pages)
```

Comment:
Chunk doesn't make empty arrays, this is all about specially massaging the `num_pages` value because `PageSelector` doesn't behave well, right? Fix `PageSelector`'s behaviour, or at least push this logic down in to that component. Otherwise, this is just another time bomb.